File: Genericprogram/Specificoop.java
import java.lang.*;

class Specificoop
{
    public static void main(String arg[])
    {
        Arithematic obj = new Arithematic(11,10);
        int ret = 0;
        ret = obj.Addition(10,11);
        System.out.println("Addition is:"+ret);
        

    }
}

class Arithematic
{
    public int No1;
    public int No2;

    public Arithematic(int a, int b)
    {
        No1 = a;
        No2 = b;
    }

    public int Addition(int a,int b)
    {
        int Ans = 0;
        Ans = No1 + No2;
        return Ans;
    }
}

File: Generic_programming_using_C++_&_Java/Specificoop.cpp
#include<iostream>
using namespace std;
class Arthematic 
{
    public:
    int No1 ,No2;

    Arthematic(int a, int b)
    {
        No1 = a;
        No2 = b;
    }
    int Addition()
    {
        int ans = 0;
        ans = No1 + No2;
        return ans;
    }
    int Substraction()
    {
        int ans = 0;
        ans = No1 - No2;
        return ans;
    }
};

int main()
{
    Arthematic obj(10,11);
    int ret = 0;
    ret = obj.Addition();
    cout<<"Substrction is:"<<ret<<"\n";
    
    ret = obj.Substraction();
    cout<<"Addition is:"<<ret<<"\n";

 return 0;   
}